src/goto-instrument/cover_basic_blocks.h

```diff
@@ -20,6 +20,33 @@ class message_handlert;
 
 class cover_basic_blockst
 {
+private:
```

Comment:
Merge the private sections at the end of the class.

Reply:
@peterschrammel Is there a reason for that? I'd prefer to have type definitions at the beginning of the class.

Reply:
My view on this is that I'd like to see the interface of the class first. Internal stuff can come at the end...
Of course, if there are type definitions that are part of the public interface then they should go first.